tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlAsyncParameterizedQueryTests.cs

```diff
@@ -234,6 +234,33 @@ public MicrosoftDataSqlClientAsyncTestsParameterizedQuery_ParamsWithoutAtSignFra
         }
     }
 
+    [NetFrameworkTest]
```

Comment:
The naming of this test class is inconsistent with other similar classes.  "Framework" should be abbreviated as "FW" in the class similar to how it is in the fixture or ConsoleMF class.